### ISSUE TITLE
Determine whether cpython-compile go is multithreadded

### ISSUE THREAD

**Issue by cwbeitel** [projectclarify/clarify#122]
Compiling the go implementation of the cbt-to-cbt sampler into cpython and calling from a python dataset model is a good way to retain our current means of organizing datasets as a source of truth. As the benefits of Go were parallelism esp. in relation to the Go BigTable client it will be necessary to (1) first confirm cpython programs are parallel in the desired way, and (2) can do so with a client object.

**Comment by Hekate1** [comment]
**Python Running parallel Go**
<img width="912" alt="Screen Shot 2020-01-28 at 2 05 46 AM" src="https://user-images.githubusercontent.com/19271017/73254393-31c10800-4173-11ea-9243-1a9dfe89a248.png">
**Python Running non-parallel Go**
<img width="912" alt="Screen Shot 2020-01-28 at 2 08 41 AM" src="https://user-images.githubusercontent.com/19271017/73254475-561ce480-4173-11ea-9d2e-1c12e48fc567.png">

This parallelization is being run on basic client objects, giving indication that parallelization can be used on complex objects as well.

**Comment by cwbeitel** [comment]
@Hekate1 Can you confirm that those process monitor snapshots were both obtained while the two different versions were running? Looks about right as long as the second didn't terminate before the process mon. was run, guessing the larger utilization shown in the second was a previous run of the first.

We can close this with a PR to exp/golang-experiments/cpython-parallel/... that includes the code you used to do the experiment.

Thanks for your contribution!

**Comment by Hekate1** [comment]
Yes both screenshots were obtained processes were currently running. The processes were naively calculating fib 100 so neither process finished before I killed it.
